<!DOCTYPE html
	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
	 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
<title>HVDRP Fragment Merger Tool: Reference Tutorial</title>
<link rel="stylesheet" type="text/css" href="/res/hvdr.css"/>
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>
<style type="text/css">
.hidden { visibility: hidden; }
.unhidden { visibility: visible; }
td { text-align : left; }
td.shade1 { background: lightblue; text-align: center; font-weight: bold; }
td.shade2 { background: #AAFFAA; }
td.shade3 { background: pink; }
fieldset { width: 95%; }
input.aminoacid1 { text-align:center; background:lightblue; color:darkslategrey; font-weight:bold; }
input.aminoacid2 { text-align:center; background:#ffff99; color:darkslategrey; font-weight:bold; }
input.nucleotide { text-align:center; }
div.instructions { font-family:monospace; }
span.compare { color:blue; }
span.tt { color:black; background:#DDFFDD; font-family:monospace; }
ul.samplelist { font-family:monospace; color:blue; }
.sample { font-family: monospace; color:blue; }
.mono { font-family: monospace; font-weight: bold; }
a:focus { outline: 0; }
em.i { font-style:oblique; font-weight:normal; text-decoration:none; }
em.b { font-style:normal; font-weight:bold; text-decoration:none; }
legend { font-weight:bold; padding:0.2em 0.5em; border:1px solid green; color:green; font-size:120%; font-family:monospace; background:lightgreen; }
</style>
</head>
<body>
<h1>HVDR Online Tools</h1>
<div class="note">Hepatitis Virus Diversity Research <a href="http://www.wits.ac.za/health/hvdrp">Programme</a> and Wits <a href="http://www.bioinf.wits.ac.za">Bioinformatics</a><br>University of the Witwatersrand</div>
<h2><img src="/res/fm.png" style="vertical-align:middle">Fragment Merger Tool</h2>

<br>
<fieldset>
<legend>Reference Tutorial</legend>

One or more reference sequences (in FASTA format) may be specified on the input page. These sequences are not used to create the merge. After the merge has been created, a multiple sequence alignment of the merged sequence with the reference sequence/s is performed, using the fast <i><a href="http://www.drive5.com/muscle/" target="_blank">muscle</a></i> program. The alignment is displayed on the output page. Loci which are conserved across all sequences (reference and merged) are indicated with a <span class="tt">|</span> character. Mismatches are indicated with a space character.

<br><br>The alignment is inteded to be used a quick check to validate the success and/or accuracy of the merge. When fragments which constitute a full-length circular genome (such as a viral genome) are specified, it is likely that the final merged sequence will not be in the standard orientation. The same scenario will occur when sequence data straddling the "start" position of a circular genome if specified. In the former scenario, the merge sequence can be slid into the correct orientation by specifying appropriate "slide" parameters on the input page. In either scenario, however, specifying a single reference sequence in the standard orientation will result in an alignment with long regions of gaps on one or either end. This can be avoided by preparing the refernce sequence to match the orientation/position of the merged sequence, or by preparing a double-length reference sequence.

<br><br>If the reference sequence file contains many sequences, more time will be required to generate the alignment. Typically, one or two reference sequences should be sufficient.

</fieldset>
<br>
</body>
</html>

